docs/pages/database-access/guides/cockroachdb-self-hosted.mdx

```diff
@@ -173,3 +199,6 @@ $ tsh db logout roach
 ## Next steps
 
 (!docs/pages/includes/database-access/guides-next-steps.mdx!)
+
+## Further reading
```

Comment:
Nit: Feels like this could just be rolled into "next steps". Or just keep the link inline as I mentioned above.